zerver/tests/test_retention.py

```diff
@@ -198,12 +198,12 @@ def test_cross_realm_messages_archiving_one_realm_expired(self) -> None:
         # Make the message expired on Zulip only:
         self._change_messages_pub_date([msg_id], timezone_now() - timedelta(ZULIP_REALM_DAYS+1))
 
-        move_expired_to_archive()
+        archived_data_info = move_expired_to_archive()
 
         self.assertEqual(ArchivedMessage.objects.count(), 1)
-        self.assertEqual(ArchivedUserMessage.objects.count(), 1)
+        self.assertEqual(ArchivedUserMessage.objects.count(), 2)
```

Comment:
Should we put a comment above this line that this should be 1, but we've temporarily broken this test as part of other work?

Reply:
I missed this spot, yes, should have put a comment here. Fixed now, thanks!